STYLE.md

```diff
@@ -179,6 +179,87 @@ For words that are both concepts and API terms (for example, a `Touched` event o
 - Connect the function to the `Class.BasePart.Touched` event.
 - Display text when players touch the part.
 
+## Reference YAML Files
+
+The reference documentation in `content/en-us/reference/engine/` uses YAML rather than Markdown. YAML is a structured format that has stricter whitespace requirements than Markdown. Pay attention to the amount of leading whitespace before an element; even a single missing space can result in invalid YAML. If you're not familiar with the format, consider using an online YAML linter to check your work.
+
+The documentation makes heavy use of the `|` character, which allows for multi-line strings. It preserves line breaks and lets you include multiple paragraphs, lists, tables, etc.
```

Comment:
One of the most important benefits of `|` over `>` for us is multi-line code blocks. See https://yaml-multiline.info/

```suggestion
The documentation makes heavy use of the `|` character, which allows for multi-line strings. It preserves line breaks and lets you include multiple paragraphs, lists, tables, multi-line code blocks, etc.
```